src/20_Components/amp-geo.html

```diff
@@ -0,0 +1,93 @@
+<!---{
+    "experiments": ["amp-geo"]
+}--->
+<!--
+  #### Introduction
+  The [amp-geo](https://www.ampproject.org/docs/reference/components/amp-geo) extension
+  provides country-level geolocation. The `amp-geo` component also provides a simple mechanism to group countries, 
+  making it easier to apply attributes to several countries at once.
```

Comment:
Request change to:

The [amp-geo](https://www.ampproject.org/docs/reference/components/amp-geo) extension
makes it possible to vary small sections of content based on an approximation of the users' country-level location, similar to the level of an ISO Country Code. The `amp-geo` component also provides a simple mechanism to group locations, making it easier to apply attributes to several geos at once.